src/test/resources/features/Steps_part2.feature

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Rename this file for a better name than steps: it is not a steps class, it is a feature file with scenarios.
Maybe a name that describe background.

Reply:
fixed with commit f8c385f1d168181e31b575f1cf3c3229e7c6c2b0